.github/workflows/release-publish.yml

```diff
@@ -0,0 +1,374 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Official Release Publishing Workflow
+# =====================================
+#
+# Purpose:
+# - Publish Docker images as convenience binaries that correspond 1:1 to a
+#   voted and approved source release of Apache Solr MCP.
+# - NOTE: Docker images are NOT the release of record; the authoritative
+#   release artifacts are the signed source tarballs published to the ASF
+#   distribution system (dist.apache.org / mirrors).
+#
+# When to run:
+# - AFTER the ASF voting process has completed successfully (minimum 72 hours
+#   and at least three +1 binding PMC votes) and the source release has been
+#   finalized/published.
+#
+# ASF Release Process (summary):
+# ------------------------------
+# 1. Release Manager creates release candidate (RC)
+# 2. RC is staged for voting (72-hour minimum voting period)
+# 3. PMC members vote on the release
+# 4. Source release is published to dist.apache.org / mirrors
+# 5. After successful vote and published source, this workflow is triggered manually
+# 6. Publishes Docker images to official registries as convenience binaries
+#
+# Prerequisites:
+# --------------
+# - Release must have passed ASF voting process
+# - Source release artifacts must be signed and available on dist.apache.org
+# - Release Manager must have necessary credentials
+#
+# Manual Trigger Required:
+# ------------------------
+# This workflow MUST be triggered manually by the Release Manager
+# after the ASF vote passes and the source release is published.
+
+name: Release Publish
+
+# Trigger this workflow manually from the GitHub UI and capture structured inputs
+# - We require the GA version (e.g., 1.0.0) and the RC tag suffix (e.g., rc1)
+# - Optional inputs help document the vote thread and enable experimental signing
+on:
+  workflow_dispatch:
+    inputs:
+        # Semantic version of the approved release (no -rc suffix)
+      release_version:
+        description: 'Release version (e.g., 1.0.0)'
+        required: true
+        type: string
+        # Which release candidate was approved (used to check out the exact tag)
```

Comment:
The indentation of the comments are confusing, should be in line with the block below